new-dti-website/components/initiative/InitiativeDisplay.tsx

```diff
@@ -0,0 +1,102 @@
+import Image, { ImageProps } from 'next/image';
+import { Inter } from 'next/font/google';
+import initiatives from './data/initiatives.json';
+import useScreenSize from '../../src/hooks/useScreenSize';
+import { LAPTOP_BREAKPOINT } from '../../src/consts';
+import { ibm_plex_mono } from '../../src/app/layout';
+
+const inter = Inter({ subsets: ['latin'] });
```

Comment:
Again, will probably be for the future when we get all the pages done, but we could also add this to the config for the future or the global css.